.changeset/clever-adults-add.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-pagerduty': minor
+---
+
+Add new 'disableChangeEvents' attribute to PagerDuty Card to hide the Change Events tab and disable fetching of change events for the PagerDuty service.
```

Comment:
```suggestion
---
'@backstage/plugin-pagerduty': patch
---

Add new `disableChangeEvents` prop to `EntityPagerDutyCard` to hide the Change Events tab and disable fetching of change events for the PagerDuty service.
```

Switching to `patch` since it's not breaking and the plugin is on 0.x